### PR TITLE
[zh] sync blog/2023/waypoint

### PR DESCRIPTION
**Pull request by justedennnnn**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/blog/2023/waypoint-proxy-made-simple/index.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
+---
+title: "Istio Ambient Waypoint 代理变得简单"
+description: Introducing the new destination oriented waypoint proxy for simplicity and scalability.
+引入新的面向目的地的 waypoint proxy 以实现简单性和可扩展性。
```
Comment (windsonsea) [inline content/zh/blog/2023/waypoint-proxy-made-simple/index.md:4]:
```suggestion
description: 为简化和可扩展性引入全新的面向目的地的 Waypoint Proxy。
```

File: content/zh/blog/2023/waypoint-proxy-made-simple/index.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
+---
+title: "Istio Ambient Waypoint 代理变得简单"
```
Comment (windsonsea) [inline content/zh/blog/2023/waypoint-proxy-made-simple/index.md:2]:
```suggestion
title: "Istio Ambient Waypoint Proxy 让一切变得简单"
```
waypoint proxy 需要全文一致，现在有多种变体：
- Waypoint 代理
- waypoint proxy
- Waypoint Proxy

File: content/zh/blog/2023/waypoint-proxy-made-simple/index.md [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+description: Introducing the new destination oriented waypoint proxy for simplicity and scalability.
+引入新的面向目的地的 waypoint proxy 以实现简单性和可扩展性。
+publishdate: 2023-03-31
+attribution: "Lin Sun (Solo.io), John Howard (Google)"
+keywords: [istio,ambient,waypoint]
+---
+
+Ambient 将 Istio 的功能分为两个不同的层，一个安全覆盖层和一个七层处理层。Waypoint Proxy 是一个可选组件，它基于 Envoy 并为其管理的工作负载处理 L7 处理。自 2022 年[首次启动环境](/zh/blog/2022/introducing-ambient-mesh/)以来，我们进行了重大更改以简化 waypoint 配置、可调试性和可扩展性。
```
Comment (windsonsea) [inline content/zh/blog/2023/waypoint-proxy-made-simple/index.md:10]:
```suggestion
Ambient 将 Istio 的功能分为两个不同的层，一个具备安全机制的 Overlay 层和一个 L7 处理层。Waypoint Proxy 是一个基于 Envoy 的可选组件，为其管理的工作负载进行 L7 处理。自 2022 年[首次发布 Ambient](/zh/blog/2022/introducing-ambient-mesh/) 以来，我们在简化 Waypoint 配置、可调试性和可扩展性方面做了许多重大变更。
```

File: content/zh/blog/2023/waypoint-proxy-made-simple/index.md [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+
+## waypoint proxy 的架构{#architecture-of-waypoint-proxies}
```
Comment (windsonsea) [inline content/zh/blog/2023/waypoint-proxy-made-simple/index.md:12]:
```suggestion
## Waypoint Proxy 的架构 {#architecture-of-waypoint-proxies}
```

File: content/zh/blog/2023/waypoint-proxy-made-simple/index.md [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+
+与 Sidecar 类似，waypoint proxy 也是基于 Envoy 的，由 Istio 动态配置以服务于您的应用程序配置。Waypoint Proxy 的独特之处在于它运行每个命名空间（默认）或每个服务帐户。通过在应用程序 pod 之外运行，waypoint proxy 可以独立于应用程序安装、升级和扩展，并降低运营成本。
```
Comment (windsonsea) [inline content/zh/blog/2023/waypoint-proxy-made-simple/index.md:14]:
```suggestion
与 Sidecar 类似，Waypoint Proxy 也是基于 Envoy 的，由 Istio 动态配置以服务于您的应用程序配置。Waypoint Proxy 的独特之处在于它按照每个命名空间（默认）或每个服务帐户来运行。通过在应用程序 Pod 之外运行，Waypoint Proxy 可以独立于应用程序安装、升级和扩展，并降低运营成本。
```

File: content/zh/blog/2023/waypoint-proxy-made-simple/index.md [hunk 6]
```diff
@@ -0,0 +1,215 @@
...
+
+{{< image width="100%"
+    link="waypoint-architecture.png"
+    caption="Waypoint architecture"
+    >}}
+
+Waypoint Proxy 是使用 Kubernetes 网关资源或有用的 `istioctl` 命令声明性地部署的：
```
Comment (windsonsea) [inline content/zh/blog/2023/waypoint-proxy-made-simple/index.md:21]:
```suggestion
Waypoint Proxy 是使用 Kubernetes Gateway 资源以声明方式部署的，也可以使用以下 `istioctl` 命令来部署：
```

File: content/zh/blog/2023/waypoint-proxy-made-simple/index.md [hunk 7]
```diff
@@ -0,0 +1,215 @@
...
+如果我们将每个命名空间扩大到 25 个部署，每个部署有 10 个 pod，每个航点部署有 2 个pod，以实现高可用性，那么数字就更加惊人了 -- waypoint 的配置发送量只需要挎包的 0.8%，如下表所示
+
+| 配置分发    |         命名空间 1              |       命名空间 2                |     总计     |
+| --------------------------- | -------------------------------- | -------------------------------- | ------------- |
+| Sidecars                    | 25 configurations * 250 sidecars | 25 configurations * 250 sidecars |    12500      |
+| Waypoints                   | 25 configurations * 2 waypoints  | 25 configurations * 2 waypoints  |     100       |
+| Waypoints / Sidecars        |              0.8%                |               0.8%               |      0.8%     |
+
+While we use namespace scoped waypoint proxies to illustrate the simplification above, the simplification is similar
+when you apply it to service account waypoint proxies.
```
Comment (windsonsea) [inline content/zh/blog/2023/waypoint-proxy-made-simple/index.md:79]: Line 78-79 漏译

**Review by windsonsea — COMMENTED** [review]
有几个普遍的问题，某些术语不翻译时可以采用一致的形式：

- Pod
- Sidecar
- Alpha
- Ambient
- Waypoint
- Waypoint Proxy

如果长句能在 col 80-100 适当折行，会让 reviewer 更容易阅读

File: content/zh/blog/2023/waypoint-proxy-made-simple/index.md [hunk 8]
```diff
@@ -0,0 +1,215 @@
...
+  - name: mesh
+    port: 15008
+    protocol: HBONE
+{{< /text >}}
+
+Istiod 将监控这些资源并自动为用户部署和管理相应的 waypoint proxy。
+
+## 将源代理配置转移到目的地代理{#shift-source-proxy-configuration-to-destination-proxy}
+
+在现有的 sidecar 架构中，大多数流量整形（例如[请求路由](/zh/docs/tasks/traffic-management/request-routing/zh)或[流量转移](/zh/docs/tasks/traffic-management/traffic-shifting/)或[故障注入](/zh/docs/tasks/traffic-management/fault-injection/)）策略由源（客户端）代理实现，而大多数安全策略由目标（服务器）代理实现。这导致了一些担忧：
```
Comment (wilsonwu) [inline content/zh/blog/2023/waypoint-proxy-made-simple/index.md:41]:
```suggestion
在现有的 Sidecar 架构中，大多数流量整形（例如[请求路由](/zh/docs/tasks/traffic-management/request-routing/zh)或[流量转移](/zh/docs/tasks/traffic-management/traffic-shifting/)或[故障注入](/zh/docs/tasks/traffic-management/fault-injection/)）策略由源（客户端）代理实现，
而大多数安全策略由目标（服务器）代理实现。这导致了一些担忧：
```

File: content/zh/blog/2023/waypoint-proxy-made-simple/index.md [hunk 9]
```diff
@@ -0,0 +1,215 @@
...
+
+* 缩放——每个源 sidecar 都需要知道关于网格中每个其他目的地的信息。这是一个多项式缩放问题。更糟糕的是，如果任何目标配置发生变化，我们需要立即通知所有 sidecar。
```
Comment (wilsonwu) [inline content/zh/blog/2023/waypoint-proxy-made-simple/index.md:43]:
```suggestion
* 缩放——每个源 Sidecar 都需要知道关于网格中每个其他目的地的信息。这是一个多项式缩放问题。
  更糟糕的是，如果任何目标配置发生变化，我们需要立即通知所有 Sidecar。
```

File: content/zh/blog/2023/waypoint-proxy-made-simple/index.md [hunk 10]
```diff
@@ -0,0 +1,215 @@
...
+* 调试——因为策略执行在客户端和服务器边车之间分开，所以在故障排除时很难理解系统的行为。
+* 混合环境——如果我们的系统不是所有客户端都是网格的一部分，我们就会得到不一致的行为。例如，非网格客户端不会遵守金丝雀部署策略，从而导致意外的流量分配。
+* 所有权和归属——理想情况下，在一个命名空间中编写的策略应该只影响在同一命名空间中运行的代理所做的工作。然而，在这个模型中，它是由每个 sidecar 分发和执行的。尽管 Istio 已围绕此约束进行设计以确保其安全，但它仍然不是最佳选择。
```
Comment (wilsonwu) [inline content/zh/blog/2023/waypoint-proxy-made-simple/index.md:46]:
```suggestion
* 所有权和归属——理想情况下，在一个命名空间中编写的策略应该只影响在同一命名空间中运行的代理所做的工作。
  然而，在这个模型中，它是由每个 Sidecar 分发和执行的。尽管 Istio 已围绕此约束进行设计以确保其安全，
  但它仍然不是最佳选择。
```

File: content/zh/blog/2023/waypoint-proxy-made-simple/index.md [hunk 11]
```diff
@@ -0,0 +1,215 @@
...
+特别是可扩展性问题，对于在大型集群中运行的用户来说是一个困扰。如果我们把它形象化，我们就可以看到新架构有多大的改进。
+
+考虑一个简单的部署，我们有 2 个命名空间，每个命名空间有 2 个（彩色编码的）部署。为 Sidecar 编程所需的 Envoy (XDS) 配置显示为圆圈：
+
+{{< image width="70%"
+    link="sidecar-config.png"
+    caption="Every sidecar has configuration about all other sidecars"
+    >}}
+
+在 sidecar 模型中，我们有 4 个工作负载，每个工作负载有 4 组配置。如果这些配置中的任何一个发生更改，则所有这些配置都需要更新。总共有 16 个配置分布。
```
Comment (wilsonwu) [inline content/zh/blog/2023/waypoint-proxy-made-simple/index.md:59]:
```suggestion
在 Sidecar 模型中，我们有 4 个工作负载，每个工作负载有 4 组配置。
如果这些配置中的任何一个发生更改，则所有这些配置都需要更新。总共有 16 个配置分布。
```

File: content/zh/blog/2023/waypoint-proxy-made-simple/index.md [hunk 12]
```diff
@@ -0,0 +1,215 @@
...
+| 配置分发    |         命名空间 1              |       命名空间 2                |     总计     |
+| --------------------------- | -------------------------------- | -------------------------------- | ------------- |
+| Sidecars                    | 25 configurations * 250 sidecars | 25 configurations * 250 sidecars |    12500      |
+| Waypoints                   | 25 configurations * 2 waypoints  | 25 configurations * 2 waypoints  |     100       |
+| Waypoints / Sidecars        |              0.8%                |               0.8%               |      0.8%     |
+
+While we use namespace scoped waypoint proxies to illustrate the simplification above, the simplification is similar
+when you apply it to service account waypoint proxies.
+
+虽然我们使用命名空间范围的 waypoint proxy 来说明上面的简化，但当您将其应用于服务帐户  waypoint proxy  时，简化是相似的。
```
Comment (wilsonwu) [inline content/zh/blog/2023/waypoint-proxy-made-simple/index.md:81]:
```suggestion
虽然我们使用命名空间范围的 waypoint proxy 来说明上面的简化，
但当您将其应用于服务帐户 waypoint proxy 时，简化是相似的。
```

File: content/zh/blog/2023/waypoint-proxy-made-simple/index.md [hunk 13]
```diff
@@ -0,0 +1,215 @@
...
+envoy://main_internal/                              HEALTHY OK            main_internal
+unix://./etc/istio/proxy/XDS                        HEALTHY OK            xds-grpc
+unix://./var/run/secrets/workload-spiffe-uds/socket HEALTHY OK            sds-grpc
+{{< /text >}}
+
+请注意，您不会获得与 revice 以外的任何服务相关的任何端点，即使您在`default` 和 `istio-system` 命名空间中还有一些其他服务。
+
+## 结束语{#wrapping-up}
+
+我们对专注于面向目的地的 waypoint proxy 的 waypoint 简化感到非常兴奋。这是朝着简化 Istio 的可用性、可扩展性和可调试性迈出的又一重要步骤，这些是 Istio 路线图上的重中之重。按照我们的[入门指南](http://preliminary.istio.io/latest/docs/ops/ambient/getting-started/)立即尝试环境 alpha 构建并体验简化的 waypoint proxy ！
```
Comment (wilsonwu) [inline content/zh/blog/2023/waypoint-proxy-made-simple/index.md:215]:
```suggestion
我们对专注于面向目的地的 waypoint proxy 的 waypoint 简化感到非常兴奋。
这是朝着简化 Istio 的可用性、可扩展性和可调试性迈出的又一重要步骤，
这些是 Istio 路线图上的重中之重。按照我们的[入门指南](http://preliminary.istio.io/latest/docs/ops/ambient/getting-started/)立即尝试环境
alpha 构建并体验简化的 waypoint proxy！
```

**Review by wilsonwu — COMMENTED** [review]
换行有助于review，中英文混排时一些专有名词的英文首字母大写会更好。

File: content/zh/blog/2023/waypoint-proxy-made-simple/index.md [hunk 14]
```diff
@@ -0,0 +1,225 @@
...
+| 配置分发    |         命名空间 1              |       命名空间 2                |     总计     |
+| --------------------------- | -------------------------------- | -------------------------------- | ------------- |
+| Sidecars                    | 25 configurations * 250 sidecars | 25 configurations * 250 sidecars |    12500      |
+| Waypoints                   | 25 configurations * 2 waypoints  | 25 configurations * 2 waypoints  |     100       |
+| Waypoints / Sidecars        |              0.8%                |               0.8%               |      0.8%     |
+
+虽然我们使用命名空间范围的 Waypoint Proxy 来说明上面的简化，
+但当您将其应用于服务帐户 Waypoint Proxy 时，简化是相似的。
+
+这种减少的配置意味着控制平面和数据平面的资源使用率（CPU、RAM 和网络带宽）较低。虽然今天的用户可以通过谨慎使用 exportToIstio 网络资源或 [Sidecar](/zh/docs/reference/config/networking/sidecar) API 看到类似的改进，但在 Ambient 模式下不再需要这样做，从而使扩展变得轻而易举。
```
Comment (windsonsea) [inline content/zh/blog/2023/waypoint-proxy-made-simple/index.md:87]:
```suggestion
这样减少配置意味着控制平面和数据平面更低的资源使用率（CPU、RAM 和网络带宽）。虽然如今的用户可以通过在 Istio 网络资源或 [Sidecar](/zh/docs/reference/config/networking/sidecar/) API 中谨慎使用 `exportTo` 来看到类似的改进，但在 Ambient 模式下不再需要这样做，这就使得扩缩容轻而易举。
```

**Review by windsonsea — COMMENTED** [review]
Please note the lint error.

File: content/zh/blog/2023/waypoint-proxy-made-simple/index.md [hunk 15]
```diff
@@ -0,0 +1,225 @@
...
+
+{{< text bash >}}
+$ istioctl proxy-config routes deploy/bookinfo-reviews-istio-waypoint
+NAME                                                    DOMAINS MATCH              VIRTUAL SERVICE
+encap                                                   *       /*
+inbound-vip|9080|http|reviews.default.svc.cluster.local *       /*                 reviews.default
+default
+{{< /text >}}
+
+回想一下，您没有 `exportTo` 在 Istio 网络资源上配置任何 Sidecar 资源或配置。但是，您确实部署了 `bookinfo-productpage` 路由以配置要路由到的入口网关 `productpage` ，但 `reviews` 路径点尚未意识到任何此类不相关的路由。
```
Comment (windsonsea) [inline content/zh/blog/2023/waypoint-proxy-made-simple/index.md:156]:
```suggestion
回想一下，您没有在 Istio 网络资源上配置任何 Sidecar 资源，也未执行 `exportTo` 配置。
然而，您部署了 `bookinfo-productpage` 路由来配置 Ingress 网关以路由到 `productpage`，
但 `reviews` Waypoint 还未感知到此类不相关的路由。
```

File: content/zh/blog/2023/waypoint-proxy-made-simple/index.md [hunk 16]
```diff
@@ -0,0 +1,225 @@
...
+
+显示路由的详细信息 `inbound-vip|9080|http|reviews.default.svc.cluster.local`，您会看到基于权重的路由配置将 90% 的流量引导到 `reviews` v1，将 10% 的流量引导到 `reviews` v2，以及 Istio 的一些默认重试和超时配置。这证实了流量和弹性策略已从源转移到目标导向 Waypoint ，如前所述。
```
Comment (windsonsea) [inline content/zh/blog/2023/waypoint-proxy-made-simple/index.md:158]:
```suggestion
在显示 `inbound-vip|9080|http|reviews.default.svc.cluster.local` 路由的详情时，
您会看到基于权重的路由配置将 90% 的流量引导到 `reviews` v1，将 10% 的流量引导到 `reviews` v2，
还能看到 Istio 的一些默认重试和超时配置。如前所述，这证实了流量和弹性策略已从来源转移到面向目的地的 Waypoint。
```

**Review by windsonsea — COMMENTED** [review]
可以在 col 80 - 120 适当折行，也方便你自己反复阅读、校对

File: content/zh/blog/2023/waypoint-proxy-made-simple/index.md [hunk 17]
```diff
@@ -0,0 +1,225 @@
...
+reviews.default.svc.cluster.local    9080 http/v1 inbound-vip EDS
+reviews.default.svc.cluster.local    9080 http/v2 inbound-vip EDS
+reviews.default.svc.cluster.local    9080 http/v3 inbound-vip EDS
+sds-grpc                             -    -       -           STATIC
+xds-grpc                             -    -       -           STATIC
+zipkin                               -    -       -           STRICT_DNS
+{{< /text >}}
+
+注意，列表中没有 `Outbound` 集群，您可以用 `istioctl proxy-config cluster deploy/bookinfo-reviews-istio-waypoint --direction outbound` 来确认!
+最棒的是，您不需要在任何其他 bookinfo 服务上配置 `exportTo`（例如，产品页或评级服务）。换句话说，reviews Waypoint 不会被告知任何不必要的集群，不需要您进行任何额外的手动配置。
```
[the remark this answers is not in the document]
Comment (justedennnnn) [reply content/zh/blog/2023/waypoint-proxy-made-simple/index.md:144]: 收到

**Review by windsonsea — APPROVED** [review]
Let's kick this in. Some minor issues can be covered later.

**Pull request merged**